staging_manifest.yml

```diff
@@ -13,9 +13,10 @@ services:
 - federalist-staging-s3
 - federalist-staging-env
 env:
-  NODE_ENV: staging
+  NODE_ENV: production
+  APP_ENV: staging
   SAILS_LOG_LEVEL: verbose
-  NPM_CONFIG_PRODUCTION: false
+  NPM_CONFIG_PRODUCTION: true
```

Comment:
Although if the buildpack ever changes, you can be sure that the `NODE_ENV` changes will need to be introduced again. So maybe it's safer to use `APP_ENV` for Sails in the long-run.